app/models/invenio_rdm_record_converter.rb

```diff
@@ -131,11 +135,11 @@ def invenio_metadata(gf)
       "description": gf.description.first,
       "additional_descriptions": additional_descriptions(gf.description),
       "publisher": gf.publisher.shift,
-      "publication_date": "#{gf.date_uploaded.year}-#{gf.date_uploaded.month}-#{gf.date_uploaded.day}",
+      "publication_date": format_publication_date(gf.date_created.shift || gf.date_uploaded.shift),
       "subjects": SUBJECT_SCHEMES.map{ |subject_type| subjects_for_scheme(gf.send(subject_type), subject_type) }.flatten,
       "contributors": contributors(gf.contributor),
       "dates": gf.date_created.map{ |date| {"date": date, "type": "other", "description": "When the item was originally created."} },
-      "languages": gf.language.any?{ |lang| lang.downcase == ENGLISH} ? ["eng"] : "",
+      "languages": gf.language.any?{ |lang| lang.downcase == ENGLISH } ? ["eng"] : "",
```

Comment:
probably best to ignore that whitespace change to sidestep a conflict with the other languags PR

Reply:
That's a good point. I'll remove that.